pallets/ddc-payouts/src/lib.rs

```diff
@@ -0,0 +1,720 @@
+//! # DDC Payouts Pallet
+//!
+//! The DDC Payouts pallet is used to distribute payouts based on DAC validation
+//!
+//! - [`Config`]
+//! - [`Call`]
+//! - [`Pallet`]
+//!
+//! ## GenesisConfig
+//!
+//! The DDC Payouts pallet depends on the [`GenesisConfig`]. The
+//! `GenesisConfig` is optional and allow to set some initial nodes in DDC.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+#![recursion_limit = "256"]
+
+use ddc_primitives::{ClusterId, DdcEra};
+use ddc_traits::{cluster::ClusterVisitor, customer::CustomerCharger as ICustomerCharger};
```

Comment:
Is there concrete type called `CustomerCharge` otherwise, why would we want to call it `ICustomerCharge`... this is not the usual coding practice.

Reply:
clippy didn't like both the name and the type of the same name, I guess it can be renamed to something else or better yet a fully qualified name